blogs/data-mover/data-mover-1.2.0-overview.md

```diff
@@ -0,0 +1,134 @@
+<h2>Introduction</h2>
+
+<p dir="auto">Data Mover provides portability and durability of CSI volume snapshots by relocating snapshots into an object storage location during backup of a stateful application. These snapshots are then available for restore during instances of disaster scenarios. This blog will discuss the different Data Mover components and how they work together to complete this process.</p>
+
+<h2>What Is CSI?</h2>
+
+<p dir="auto">One of the more important components of Data Mover to understand is CSI, or Container Storage Interface. CSI provides a layer of abstraction between container orchestration tools and storage systems such that users do not need to be informed on the differences between storage provider's needs and requirements. It also provides point-in-time snapshotting of volumes.</p>
+
+<p dir="auto">CSI volumes are now the industry standard and are the storage backing for most Cloud Native applications. However, issues concerning CSI volumes still remain. Some volumes have vendor-specific requirements, and can prevent proper portability and durability. Data Mover works to solve this case, which will be discussed more in the next section.</p>
```

Comment:
```suggestion
<p dir="auto">CSI compliant storage plugins are now the industry standard and are the preferred storage plugin type for most Cloud Native applications including Kubernetes. However, issues concerning CSI volumes still remain. Some volumes have vendor-specific requirements, and can prevent proper portability and durability. Data Mover works to solve this case, which will be discussed more in the next section.</p>
```

Reply:
src for in-tree driver deprecation: https://kubernetes.io/blog/2022/09/26/storage-in-tree-to-csi-migration-status-update-1.25/#timeline-and-status